modules/021-cni-cilium/monitoring/prometheus-rules/agent.yaml

```diff
@@ -0,0 +1,89 @@
+- name: cni-cilium
+  rules:
+  - alert: CiliumAgentControllersRunsNotSucceeding
+    expr: max by (pod,namespace) (cilium_controllers_failing) > 0
+    for: 15m
+    labels:
+      d8_module: cni-cilium
+      d8_component: agent
+    annotations:
+      plk_protocol_version: "1"
+      plk_markup_format: "markdown"
+      plk_grouped_by__main: CiliumAgentMalfunctioning,prometheus=deckhouse,tier=~tier
+      plk_create_group_if_not_exists__main: CiliumAgentMalfunctioning,prometheus=deckhouse,tier=~tier
+      summary: Some controllers (internal logical loops) are failing in agent `{{`{{ $labels.namespace }}`}}`/`{{`{{ $labels.pod }}`}}`
+      description: |
+        Check what's going on: `kubectl -n `{{`{{ $labels.namespace }}`}}` logs `{{`{{ $labels.pod }}`}}``
+
+  - alert: CiliumAgentUnreachableNodes
+    expr: max by (namespace, pod) (cilium_unreachable_nodes) > 0
+    for: 5m
+    labels:
+      d8_module: cni-cilium
+      d8_component: agent
+    annotations:
+      plk_protocol_version: "1"
+      plk_markup_format: "markdown"
+      plk_grouped_by__main: CiliumAgentMalfunctioning,prometheus=deckhouse,tier=~tier
+      plk_create_group_if_not_exists__main: CiliumAgentMalfunctioning,prometheus=deckhouse,tier=~tier
+      summary: Some nodes are not reachable by agent `{{`{{ $labels.namespace }}`}}`/`{{`{{ $labels.pod }}`}}`.
+      description: |
+        Check what's going on: `kubectl -n `{{`{{ $labels.namespace }}`}}` logs `{{`{{ $labels.pod }}`}}``
+
+  - alert: CiliumAgentUnreachableHealthEndpoints
+    expr: max by (namespace, pod) (cilium_unreachable_health_endpoints) > 0
+    for: 5m
+    labels:
+      d8_module: cni-cilium
+      d8_component: agent
+    annotations:
+      plk_protocol_version: "1"
+      plk_markup_format: "markdown"
+      plk_grouped_by__main: CiliumAgentMalfunctioning,prometheus=deckhouse,tier=~tier
+      plk_create_group_if_not_exists__main: CiliumAgentMalfunctioning,prometheus=deckhouse,tier=~tier
+      summary: Some nodes' health endpoints are not reachable by agent `{{`{{ $labels.namespace }}`}}`/`{{`{{ $labels.pod }}`}}`.
```

Comment:
```suggestion
      summary: Some nodes health endpoints are not reachable by agent `{{`{{ $labels.namespace }}`}}`/`{{`{{ $labels.pod }}`}}`.
```

Reply:
That is not a mistake. It's plural.

![image](https://user-images.githubusercontent.com/950313/171580059-7ebc9d67-579f-4ea5-9935-95d516b25601.png)